chromium/pages/cancel/index.html

```diff
@@ -8,11 +8,18 @@
     <link rel="stylesheet" href="style.css" type="text/css" />
   </head>
   <body>
-    <h1 id="https-everywhere"><img src="../../images/banner-red.png" alt="HTTPS Everywhere" /></h1>
-    <p data-i18n="cancel_he_blocking_explainer"></p>
-    <p id="url-paragraph"><span id="url-label">URL: </span><a href="#" id="originURL"></a></p>
-    <script src="../translation.js"></script>
-    <script src="../util.js"></script>
-    <script src="ux.js"></script>
+    <div class="content">
+      <h1 id="https-everywhere"><img src="../../images/banner-red.png" alt="HTTPS Everywhere" /></h1>
+      <p data-i18n="cancel_he_blocking_explainer"></p>
+      <p id="url-paragraph">
+        <span id="url-label">URL: </span><a href="#" id="url-value">PLACEHOLDER</a>
+      </p>
+      <form method="get" action="#" id="url-form">
```

Comment:
no need for `<form>` here.

Reply:
Are you wanting an anchor element instead of the button (and form)?

Reply:
@patch-malone `<button>` doesn't need a form.

Reply:
What about the button's `formaction`? I think it's better to persist the URL in the standard HTML attribute (`formaction`) on `<button>` itself rather than dynamically reading it from another part of the page because that will make it clear to someone inspecting source where the button takes them, along with making the value available to screenreaders.

Reply:
I can understand hiding the internal extension URL though. What do you recommend?

Reply:
Change the button to `<a>` then. That would make more sense in this case.

Reply:
The `<button>` has been changed to an `<a>` tag.